pl_engine/pl_server/src/main/java/com/cubrid/plcsql/compiler/PlcsqlCompilerMain.java

```diff
@@ -84,6 +84,26 @@ public static CompileInfo compilePLCSQL(
     private static final int OPT_VERBOSE = 1;
     private static final int OPT_PRINT_PARSE_TREE = 1 << 1;
 
+    private static String cutExpectingClause(String errMsg) {
+
+        int idx;
+        if (errMsg != null && (idx = errMsg.lastIndexOf(" expecting ")) > 0) {
+
+            String tail = errMsg.substring(idx + 11); // 11: length of " expecting "
+
+            if (tail.matches("[A-Z0-9_]+") /* single token name */
+                    || (tail.startsWith("'")
+                            && tail.endsWith("'")) /* single token of the form '...' */
+                    || (tail.startsWith("{")
+                            && tail.endsWith("}") /* multiple tokens of the form {...} */)) {
```

Comment:
```suggestion
                    || (tail.startsWith("'") && tail.endsWith("'")) /* single token of the form '...' */
                    || (tail.startsWith("{") && tail.endsWith("}") /* multiple tokens of the form {...} */)) {
```
이렇게 하는게 더 직관적인 느낌이지만 큰 의미는 없습니다. 편하신 대로 하세요~

Reply:
저도 그렇게 하고 싶으나 code indentation 툴을 돌리니 저렇게 바뀌네요.